release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -469,6 +469,12 @@ The following metric has been removed:
 
 * `ovnkube_master_skipped_nbctl_daemon_total`
 
+[id="ocp-4-12-multi-zone-ipi-vsphere-installation"]
+==== Multi-zone IPI VMware vSphere installation
+Beginning with {product-title} {product-version}, you can configure multiple datacenter, single vCenter installations using IPI installation.
+
+For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-zones.adoc[Installing a cluster on vSphere with Multiple Failure Domains].
```

Comment:
```suggestion
// This link will be added in post-GA
For more information, see xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-zones.adoc[Installing a cluster on vSphere with Multiple Failure Domains].
```

Reply:
Done, thank you!

Reply:
@opayne1 I had to remove the link completely, Travis kept "seeing" it even when I commented out. So I removed the "For more information,..." but left the note about the link being added in post-GA.